### PR TITLE
i need help i 3rd test

### PR DESCRIPTION
**Pull request by Serhii-Hrynyshyn**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,13 +1,72 @@
...
+                    "source": "flake8"
+                },
+                {
+                    "line": errors[1]["line_number"],
+                    "column": errors[1]["column_number"],
+                    "message": errors[1]["text"],
+                    "name": errors[1]["code"],
+                    "source": "flake8"
+                }
+            ],
```
Comment (ArsenPidhoretskyi) [inline app/main.py]: Not true use list comprehension and use format_linter_error

File: app/main.py [hunk 2]
```diff
@@ -1,13 +1,72 @@
...
+                        "message": errors_linter[1][1][
+                            "text"],
+                        "name": errors_linter[1][1][
+                            "code"],
+                        "source": "flake8"
+                    }
+                ],
+            "path": "./source_code_2.py",
+            "status": "failed"
+        }
```
Comment (ArsenPidhoretskyi) [inline app/main.py]: Total hardcode..... Use comprehensions and reuse functions

File: tests/test_main.py
Comment (ArsenPidhoretskyi) [inline tests/test_main.py:1]: Restore this code

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]

File: tests/test_main.py
Comment (NikOneZ1) [inline tests/test_main.py:1]: Restore this file, please

**Review by NikOneZ1 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**